test/test_loss_graphs.py

```diff
@@ -30,26 +30,18 @@ def test_rmse_dense_loss_biased(self):
         model = TensorRec(loss_graph=RMSEDenseLossGraph, biased=True)
         model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
 
-    def test_separation_loss(self):
-        model = TensorRec(loss_graph=SeparationLossGraph)
-        model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
-
-    def test_separation_loss_biased(self):
-        model = TensorRec(loss_graph=SeparationLossGraph, biased=True)
-        model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
-
     def test_wmrb_loss(self):
         model = TensorRec(loss_graph=WMRBLossGraph)
-        model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
+        model.fit(self.interactions, self.user_features, self.item_features, epochs=5, n_sampled_items=10)
 
     def test_wmrb_loss_biased(self):
         model = TensorRec(loss_graph=WMRBLossGraph, biased=True)
-        model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
+        model.fit(self.interactions, self.user_features, self.item_features, epochs=5, n_sampled_items=10)
 
     def test_wmrb_alignment_loss(self):
         model = TensorRec(loss_graph=WMRBAlignmentLossGraph)
-        model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
+        model.fit(self.interactions, self.user_features, self.item_features, epochs=5, n_sampled_items=10)
 
     def test_wmrb_alignment_loss_biased(self):
         model = TensorRec(loss_graph=WMRBAlignmentLossGraph, biased=True)
-        model.fit(self.interactions, self.user_features, self.item_features, epochs=5)
+        model.fit(self.interactions, self.user_features, self.item_features, epochs=5, n_sampled_items=10)
```

Comment:
maybe add another test with different n_sampled_items? Make sure that it changes at least?

Reply:
Good point -- I think a test of the `n_random_samples` should live in `TensorRecTestCase`, since the logic is contained in the `TensorRec` object.

At the very least, these loss graph tests should do more than just make sure the model fits.